mri/reconstructors/utils/extract_sensitivity_maps.py

```diff
@@ -60,44 +63,67 @@ def extract_k_space_center_and_locations(data_values, samples_locations,
         if img_shape is None:
             raise ValueError('target image cartesian image shape must be fill')
         raise NotImplementedError
+    if data_values.ndim > 2:
+        warnings.warn('Data Values seem to have rank '
+                      + str(data_values.ndim) +
+                      ' (>2). Using is_fft for now.')
+        is_fft = True
+    if is_fft:
+        img_shape = np.asarray(data_values[0].shape)
+        mask = convert_locations_to_mask(samples_locations, img_shape)
+        indices = np.where(np.reshape(mask, mask.size))[0]
+        data_ordered = np.asarray([
+            np.reshape(data_values[channel], mask.size)[indices]
+            for channel in range(data_values.shape[0])])
     else:
```

Comment:
@paquiteau could you add back this `else` (even if unnecessary): it will make reviewing much much easier for me

Reply:
Actually it's ok I though I had "Hide whitespace changes" activated, but I did not, so it should be fine